dmutils/content_loader.py

```diff
@@ -387,6 +387,30 @@ def _get_single_question_data(self, form_data):
 
         if self.id == 'serviceTypes' or self.type in ['list', 'checkboxes']:
             value = form_data.getlist(self.id)
+        elif self.type == 'boolean_list':
+            value = []
+
+            # if self.id is 'q5', form values will come back as 'q5-0', 'q5-1', 'q5-3', ...
+            # here, we build a list of the values after the hyphen, eg [0, 1, 3, ...]
+            boolean_list_indexes = [
+                int(key.split('-')[-1]) for key in form_data.keys()
+                if key.startswith("{}-".format(self.id)) and
+                key.split('-')[-1].isdigit()
+            ]
+
+            if not len(boolean_list_indexes):
+                return {}
+
+            # we don't need each value in the list of indexes, we just want the max value
```

Comment:
The wording of this comment could just be something like: `# check for values for all keys up to the max index`

It's a shame there's no way to do anything at this point for missing responses beyond the max index, but I guess we just need to rely on the API complaining that the list is too short.